people.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
- [ ] Kindly delete this file because it seems to be a duplicate file of what is already in the data folder.

Reply:
@t-yanick , thanks for your feedback, on it..